downstream/modules/platform/ref-images-inventory-variables.adoc

```diff
@@ -2,64 +2,93 @@
 
 = Image variables
 
-[cols="50%,50%,50%",options="header"]
-|====
-| *RPM variable name* | *Container variable name* | *Description*
-| | `controller_image` | {ControllerNameStart} image. 
-
-Default = `controller-rhel8:latest`
-
-| | `de_extra_images` | Decision environment extra images. 
-| | `de_supported_image` | Decision environment supported image. 
-
-Default = `de-supported-rhel8:latest`
-
-| | `eda_image` | {EDAName} image. 
-
-Default = `eda-controller-rhel8:latest`
-
-| | `eda_web_image` | {EDAName} web image. 
-
-Default = `eda-controller-ui-rhel8:latest`
-| | `ee_29_enabled` | Enable {ExecEnvShort} 29. 
-
-Default = `false`
-
-| | `ee_29_image` | Execution environment 29 image. 
-
-Default = `ee-29-rhel8:latest`
-
-| | `ee_extra_images` | Execution environment extra images. 
-| | `ee_minimal_image` | Execution environment minimal image. 
-
-Default = `ee-minimal-rhel8:latest`
-
-| | `ee_supported_image` | Execution environment supported image. 
-
-Default = `ee-supported-rhel8:latest`
-
-| | `hub_image` | {HubNameStart} image. 
-
-Default = `hub-rhel8:latest`
-
-| | `hub_web_image` | {HubNameStart} web image. 
-
-Default = `hub-web-rhel8:latest`
-
-| | `postgresql_image` | PostgreSQL image. 
-
-Default = `postgresql-15:latest`
-
-| | `receptor_image` | Receptor image. 
-
-Default = `receptor-rhel8:latest`
-
-| | `redis_image` | Redis image. 
-
-Default = `redis-6:latest`
-
-| | `pcp_image` | Performance Co-Pilot image. 
-
-Default = `rhel8-pcp:latest`
-
-|====
\ No newline at end of file
+[cols="25%,25%,30%,10%,10%",options="header"]
+|===
+| RPM variable name | Container variable name | Description | Required or optional | Default
+
+| 
+| `controller_image` 
+| Container image for {ControllerName}.
+| Optional
+| `controller-rhel8:latest`
+
+|
+| `de_extra_images` 
```

Comment:
In RPM installer, `extra_images` are used for any additional images, regardless of DEs/EEs or whatever else.